test/azul_test_case.py

```diff
@@ -56,6 +59,9 @@ class AzulTestCase(TestCase):
     def setUpClass(cls) -> None:
         super().setUpClass()
 
+        cls.aws_account_id = config.aws_account_id
```

Comment:
```suggestion
        cls.aws_account_id = os.environ['AZUL_AWS_ACCOUNT_ID']
```

For symmetry. There is no point in using the abstraction if the line below does not. The long term solution would be to extend the abstraction to handle patches like this.